db/schema.rb

```diff
@@ -39,7 +39,7 @@
     t.index ["attachment_id"], name: "index_attachment_sources_on_attachment_id"
   end
 
-  create_table "attachments", id: :integer, charset: "utf8mb3", collation: "utf8_unicode_ci", force: :cascade do |t|
+  create_table "attachments", id: :integer, charset: "utf8mb3", collation: "utf8mb3_unicode_ci", force: :cascade do |t|
```

Comment:
Did you run `govuk-docker-run bundle exec rails:migrate` or ` bundle exec rails:migrate` 

If you did the latter and have a different version of mysql locally it might have created these changes

Reply:
I ran `govuk-docker-run bundle exec rails db:migrate`, not sure where this came from